pcsx2/SIO/Memcard/MemoryCardFolder.h

```diff
@@ -26,6 +26,10 @@
 
 //#define DEBUG_WRITE_FOLDER_CARD_IN_MEMORY_TO_FILE_ON_CHANGE
 
+#ifdef _WIN32
+#define timegm _mkgmtime
```

Comment:
Please don't define macros like this in headers. I spent way too long removing them from old wx code, it just creates headaches when all of a sudden another file fails to compile when you remove an indirect include.

Do `#ifdef _MSC_VER` in the source file instead.

Reply:
Moved the macro and redid all my tests again.

Reply:
Sorry, perhaps I wasn't clear enough - this doesn't need a macro. Just put the call site in an ifdef, there's only one place.

Reply:
Okay, hope I got it right this time.
Want me to squash the commits into 1 or will that be done automatically during merge?